utils/ansible_galaxy_readme_template.md

```diff
@@ -3,6 +3,9 @@
 
 @DESCRIPTION@
 
+The tasks that are used in this role are generated using OpenSCAP.
+See the OpenSCAP project for more details at [https://github.com/OpenSCAP/openscap](https://github.com/OpenSCAP/openscap)
```

Comment:
Should ComplianceAsCode be mentioned here instead?

e.g. ``Tasks used in this role are generated from the ComplianceAsCode Project. For more details, see https://github.com/ComplianceAsCode/content``

Reply:
@shawndwells I added both. People need to understand that the tasks are generated using OpenSCAP, but that ComplianceAsCode is where to submit fixes.